index.html

```diff
@@ -981,6 +981,29 @@ <h2>A project for first-time contributions</h2>
     </div>
     <!-- ________ Maira's card END ________  -->
 
+        <!-- ________ SaiKumar's card START ________  -->
+        <div class="card">
+
+            <p class="name">SaiKumar Immadi</p>
+            <p class="contact"><i class="fa fa-github"></i> <a href="https://github.com/s-xync" target="_blank">SaiKumar on GitHub</a></p>
+            <p class="about">I want to be a Machine-Learning Engineer one day. Right now, I am just another Undergraduate Computer Science student from India who will hopefully be graduated by 2019 from Indian Institute of Information Technology Guwahati.</p>
+            <div class="resources">
+                <p>3 Useful Dev Resources</p>
+                <ul>
+                    <li><a href="#" target="_blank" title="https://www.reddit.com/r/MachineLearning/">/r/MachineLearning/</a></li>
+                    <li><a href="#" target="_blank" title="https://atom.io/">Atom Text Editor</a></li>
+                    <li><a href="#" target="_blank" title="https://in.udacity.com/course/how-to-use-git-and-github--ud775">How to Use Git and GitHub | Udacity</a></li>
```

Comment:
You placed the links in the `title` but they need to replace the `#` in the `href` attribute instead. The `title` is for the tooltip; a brief description that appears on mouse hover.